src/popups/ConfirmWindow/ConfirmWindow.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { confirmable } from 'react-confirm';
+import classNames from 'classnames';
+
+import styles from './ConfirmWindow.module.css';
+
+export const ConfirmWindow = confirmable(({ show, proceed, conformation }) => {
+    return (
+        <div
+            className={classNames(styles.windowContainer, {
+                [styles.notActive]: !show,
+            })}
+        >
+            <div className={styles.window}>
```

Comment:
Window again? ... This already starts being confusing :)
Maybe `popupContentContainer`?